File: OpenCV/02_adquisicion/cpp/main.cpp
#include <opencv2/opencv.hpp>
#include <stdexcept>
int main(){ cv::VideoCapture cap(0); if(!cap.isOpened()) throw std::runtime_error("No se pudo abrir la cámara"); cv::Mat frame; while(cap.read(frame)){ cv::imshow("camara",frame); int k=cv::waitKey(1); if(k==27 || k=='q') break; } }

File: OpenCV/02_adquisicion/python/main.py
import cv2
cap = cv2.VideoCapture(0)
if not cap.isOpened(): raise RuntimeError("No se pudo abrir la cámara")
while True:
    ok, frame = cap.read()
    if not ok: break
    cv2.imshow("camara", frame)
    if cv2.waitKey(1) & 0xFF in (27, ord('q')): break
cap.release(); cv2.destroyAllWindows()
